lib/ClangImporter/ImportType.cpp

```diff
@@ -2015,9 +2015,6 @@ ImportedType ClangImporter::Implementation::importMethodType(
       continue;
     }
 
-    if (kind == SpecialMethodKind::NSDictionarySubscriptGetter)
-      nonNullArgs.empty();
```

Comment:
Oh no. :-( RIP, code that was trying to call `clear()` but never did.

Reply:
Heh, yeah. This isn't actually needed for this PR, but happened to be in my local tree.

Reply:
Too late to enable it now ;)